example/multistep/als.py

```diff
@@ -0,0 +1,71 @@
+import click
+
+import mlflow
+import mlflow.spark
+
+import pyspark
+from pyspark.ml import Pipeline
+from pyspark.ml.recommendation import ALS
+from pyspark.ml.evaluation import RegressionEvaluator
+
+
+@click.group()
+def cli():
+    pass
```

Comment:
I don't think you'll need a group if there's only one command in Click. I think your main could just call train_als() and it will do the right thing with click.command() on top of it.